gemini/sample-apps/finance-advisor-spanner/pages/1_Asset_Search.py

```diff
@@ -0,0 +1,174 @@
+        data = return_vals.get("data")
+
+        with st.expander("Spanner Query"):
+            with stylable_container(
+                "codeblock",
+                """
+            code {
+                white-space: pre-wrap !important;
+            }
+            """,
+            ):
+                st.code(spanner_query, language="sql", line_numbers=False)
+
+        # st.success('Done!')
+    # formatted_time = f"{time_spent:.3f}"  # f-string for formatted output
+    # st.text(f"The Query took {formatted_time} seconds to complete.")
+
+    # data_load_state = st.text('Loading data...')
+    #   data_load_state.text('Loading data...done!')
+    interactive_table(data, caption="", **it_args)
+
+
+def asset_search():
+    """This function immplements Asset Search"""
+    # st.image('images/Finvest-white-removebg-small.png')
+    st.header("FinVest Fund Advisor")
+    st.subheader("Asset Search")
+
+    classes_col, buttons_col, style_col, render_with_col = st.columns(
+        [0.25, 0.25, 0.20, 0.10]
+    )
+    classes = ["display", "compact", "cell-border", "stripe"]
+    buttons = ["pageLength", "csvHtml5", "excelHtml5", "colvis"]
+    # render_with = "itables"
+    style = "table-layout:auto;width:auto;margin:auto;caption-side:bottom"
+    it_args = dict(
+        classes=classes,
+        style=style,
+    )
+
+    if buttons:
+        it_args["buttons"] = buttons
+
+    # st.subheader('Funds Matching your Search')
+    query_params = []
+    query_params.append(investment_strategy)
+    query_params.append(investment_manager)
+
+    with st.spinner("Querying Spanner..."):
+        time.sleep(1)
+        # start_time = time.time()
+
+        return_vals = fts_query(query_params)
+        spanner_query = return_vals.get("query")
+        # time_spent = time.time() - start_time
+        data = return_vals.get("data")
+
+        with st.expander("Spanner Query"):
+            with stylable_container(
+                "codeblock",
+                """
+            code {
+                white-space: pre-wrap !important;
+            }
+            """,
+            ):
+                st.code(spanner_query, language="sql", line_numbers=False)
+
+        # st.success('Done!')
+    # formatted_time = f"{time_spent:.3f}"  # f-string for formatted output
+    # st.text(f"The Query took {formatted_time} seconds to complete.")
+
+    # data_load_state = st.text('Loading data...')
+    #   data_load_state.text('Loading data...done!')
+    interactive_table(data, caption="", **it_args)
+
+
+with st.sidebar:
+    with st.form("Asset Search"):
+        st.subheader("Search Criteria")
+        preciseVsText = st.radio("", ["Full-Text", "Precise"], horizontal=True)
+        precise_search = False
+        with st.expander("Asset Strategy", expanded=True):
+            investment_strategy_pt1 = st.text_input("", value="Europe")
+            andOrExclude = st.radio("", ["AND", "OR", "EXCLUDE"], horizontal=True)
```

Comment:
Use pythonic variable names throughout
```suggestion
            and_or_exclude = st.radio("", ["AND", "OR", "EXCLUDE"], horizontal=True)
```

Reply:
done